ksql-engine/src/main/java/io/confluent/ksql/codegen/SqlToJavaVisitor.java

```diff
@@ -483,24 +485,24 @@ private String visitBooleanComparisonExpression(ComparisonExpression.Type type)
         if (paternString.endsWith("%")) {
           return new Pair<>(
               "(" + valueString + ").contains(\""
-              + paternString.substring(1, paternString.length() - 1)
-              + "\")",
-              Schema.STRING_SCHEMA
+                  + paternString.substring(1, paternString.length() - 1)
```

Comment:
nit: whitespace changes? elsewhere too

Reply:
Seems that IntelliJ adds the spaces based on our style. The two parameters for `Pair` have the same indentation but since the first parameter is broken into multiple lines the second and third line have spaces to distinguish them from the parameters.